pkg/kubernetes/deploy.js

```diff
@@ -73,6 +80,220 @@ define([
         $("#deploy-app-dialog").dialog("wait", promise);
     }
 
+
+    function run_nulecule() {
+        $("#deploy-app-dialog").dialog("failure", null);
+        var promise = validate_app_fields()
+            .fail(function(exs) {
+                $("#deploy-app-dialog").dialog("failure", exs);
+            })
+            .done(function(fields) {
+
+                nulecule_client.kill_atomicapp();
+                var promise1 = nulecule_client.writeAnswerfile(install_dir, nulecule_client.get_answers())
```

Comment:
Suggest calling it 'write_answer_file' for consistency. Also why do we need to pass in the answers? Doesn't the client know the answers to write? Or have you done it this way for testability purposes?